scripts/treedb_vectordbbench_artifact_test.py

```
@@ -299,6 +405,225 @@ def test_route_proof_skip_requires_real_load_evidence(self) -> None:
+            manifest, events = lifecycle_fixture(root)
+            events.pop(5)
+            rewrite_lifecycle_fixture(root, manifest, events)
+            with contextlib.redirect_stdout(io.StringIO()):
+                malformed_complete = harness.main([
+                    "--validate-lifecycle", str(root), "--allow-partial",
+                ])
+
+        self.assertEqual(malformed_complete, 1)
+
+        with tempfile.TemporaryDirectory() as tmp:
+            with contextlib.redirect_stdout(io.StringIO()):
```

Ruff (0.16.2)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)